perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/processing/AuditerListener.java

```diff
@@ -39,18 +39,18 @@
 
 	private final static Logger log = LoggerFactory.getLogger(AuditerListener.class);
 
-	private EventQueue eventQueue;
+	private LinkedBlockingQueue<Event> eventQueue;
```

Comment:
The type of the variable should be BlockingQueue, i.e. more general, in the same way as List (general interface) is preferable to ArrayList (concrete implementation of the interface).